inttest/common/footloosesuite.go

```diff
@@ -105,10 +104,7 @@ type FootlooseSuite struct {
 	ControllerNetworks           []string
 	WorkerNetworks               []string
 
-	/* context and cancellation */
-
-	ctxPtr atomic.Pointer[suiteCtx]
-
+	sctx suiteCtx
```

Comment:
Now that the atomic isn't required anymore, we could potentially simply inline the two variables from the suiteCtx struct here.